packages/shorebird_cli/lib/src/artifact_builder.dart

```diff
@@ -547,6 +548,53 @@ Either run `flutter pub get` manually, or follow the steps in ${cannotRunInVSCod
     return File(outFilePath);
   }
 
+  /// Builds a windows app and returns the x64 Release directory
+  Future<Directory> buildWindowsApp({
+    String? flavor,
+    String? target,
+    List<String> args = const [],
+    String? base64PublicKey,
+    DetailProgress? buildProgress,
+  }) async {
+    await _runShorebirdBuildCommand(() async {
+      const executable = 'flutter';
+      final arguments = [
+        'build',
+        'windows',
+        '--release',
+        ...args,
+      ];
+
+      final buildProcess = await process.start(
+        executable,
+        arguments,
+        runInShell: true,
+        // TODO(bryanoltman): support this
+        // environment: base64PublicKey?.toPublicKeyEnv(),
```

Comment:
Do we need a list of remaining windows bugs somewhere?

Reply:
Not a bad idea, started one at https://github.com/shorebirdtech/shorebird/issues/2745